src/track/bpm.h

```diff
@@ -12,6 +12,8 @@ class Bpm final {
 public:
     static constexpr double kValueUndefined = 0.0;
     static constexpr double kValueMin = 0.0; // lower bound (exclusive)
+    static constexpr double kValueMinForTwoDecimals = 5000.0;
+    static constexpr double kValueMinForOneDecimal = 500.0;
```

Comment:
Ok. sounds good to me.  (200 would be a bit on the edge, I have a few 190BPM tracks).